presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -328,6 +336,14 @@ void PrestoServer::run() {
           proxygen::ResponseHandler* downstream) {
         server->reportServerInfo(downstream);
       });
+  httpServer_->registerGet(
```

Comment:
It seems like this will always be registered irrespective of whether you are using Prometheus. Is that desired ? We could avoid it in the Meta environment where they are not using Prometheus.